three-decalgeometry/build.boot

```diff
@@ -0,0 +1,30 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "0.1.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom {:project     'cljsjs/three-decalgeometry
+      :version     +version+
+      :description "Decal Geometry for three.js"
+      :url         "https://github.com/spite/THREE.DecalGeometry"
+      :scm         {:url "https://github.com/spite/THREE.DecalGeometry"}
+      :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download  :url      "https://raw.githubusercontent.com/spite/THREE.DecalGeometry/546cebafebd34b4b10cc47c2d6412310e6372159/src/THREE.DecalGeometry.js"
```

Comment:
If you absolutely have to use sha1 to download lib, at least define the sha in a var next to lib version and add a note about having to update both when updating package... Though as this lib doesn't use tags and doesn't have a version number, it looks like this is not meant for global use.